docs/snippets/angular/button-story-decorator.ts.mdx

```diff
@@ -1,30 +1,25 @@
 ```ts
 // button.stories.ts
 
-import { Meta, Story, componentWrapperDecorator, moduleMetadata } from '@storybook/angular';
+import { componentWrapperDecorator, moduleMetadata } from '@storybook/angular';
 
-import { ButtonComponent } from './button.component';
-import { ParentComponent } from './parent.component'; // ParentComponent contains ng-content
+import Button from './button.component';
```

Comment:
we have a near ban on default exports in the codebase (aside from CSF! 🤦 ) and we should do the same in our snippets and only use named exports

Reply:
Ok, I'll re-check the rest of the snippets and adjust accordingly. I'll push a commit in a bit.

Reply:
With the latest commit, I've unified the snippets based on your feedback. It reads better this way.